pages/linux/xvfb-run.md

```diff
@@ -0,0 +1,16 @@
+# xvfb-run
+
+> Run a command in a virtual X server environment.
+> More information: <https://www.x.org/wiki/>.
+
+- Run the specified command in a virtual X server:
+
+`xvfb-run {{command}}`
+
+- Try to get a free server number, if the default (99) is not available:
+
+`xvfb-run --auto-servernum {{command}}`
+
+- Pass arguments to the Xvfb server:
+
+`xvfb-run --server-args "{{-screen 0 1024x768x24}}"`
```

Comment:
Did you forget the `{{command}}` on the end of this one?

Reply:
Oh right, thank you!